benefits/eligibility/forms.py

```diff
@@ -27,6 +27,13 @@ def __init__(self, agency: models.TransitAgency, *args, **kwargs):
         choices = [(v.id, _(v.selection_label)) for v in agency.eligibility_verifiers.all()]
         self.fields["verifier"].choices = choices
 
+        choice_descriptions = []
+        for verifier in agency.eligibility_verifiers.all():
+            if verifier.selection_label_description:
+                choice_descriptions.append((verifier.id, _(verifier.selection_label_description)))
+
```

Comment:
This could also be done in a list comprehension, like above for `choices`

Reply:
And looking at how this is used in the `RadioSelect` widget, it seems like a `dict` would be the more appropriate data structure? `{verifier.id: _(verifier.selection_label_description)}` and then you can lookup by `id` later, vs. having to loop over the whole thing.